### DISCUSSION TITLE
mono channel

### DISCUSSION THREAD

**Discussion by luisete79** [MiczFlor/RPi-Jukebox-RFID#2277] [Q&A] [answered]
Hi all,
I am sorry if what I ask is too evident, but I have not found any explanation.
I'm building my first Phoniebox. My box is rather small and I only have room for
A) 2 shitty 3-W speakers (USB-powered) with really bad sound quality or 
B) 1 speaker a bit bigger with better quality.
I'd rather choose option B, but I have some doubts.

1) Is there any option in the web-app to mix the two channels and have a mono output? If not, would it be possible to do it via software.
2) If I have to do it phisically, I guess that it would not be as easy as to solder the two channels to the same speaker, for what I've read I would need some resistors. Does anyone have details on how to do this and where to place them?
Thanks before hands for your help, and to MiczFlor for this splendid project.
Cheers.

**Comment by pabera** [comment]
Welcome to Phoniebox!

If you have the space for a sound card, I'd recommend HifiBerry. Works out of the box with A, and be [software configured](https://www.hifiberry.com/docs/software/dual-mono-setup/) for B.

If you chose the [future3](https://github.com/MiczFlor/RPi-Jukebox-RFID/tree/future3/develop) version, you can also follow [this guide](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/components/soundcards/hifiberry.md) to install a Hifiberry sound card.

Let us know how it goes!

**Reply by luisete79** [reply]
Thanks a lot, the idea was to keep it as simple (cheap) as possible for the moment and see how it goes, to expand it in the future with sound card, etc...

**Comment by AlvinSchiller** [comment, marked as answer]
For mono downmix have a look here:

Version 2: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Troubleshooting-FAQ#use-the-phoniebox-in-mono-not-in-stereo

Version 3:
Run the audio configuration tool 
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/audio.md#developer-information

**Reply by luisete79** [reply]
Thanks a lot, that's exactly what I was looking for. I could not find it in discussions nor in wiki. Did not check the FAQ, sorry.

**Reply by luisete79** [reply]
Hi all. I finally got some time to check this. The link for version 2 works indeed. Just one comment: when I tried to copy and paste what it is written in the FAQ, nothing worked. I realised later that the quotes should be written as "", and not „“.

**Reply by s-martin** [reply]
Thanks for the feedback. I fixed the quotes.
Please mark the reply as an answer, if you like.

**Reply by yuvke** [reply]
@luisete79  I know this is not directly related, but any chance you can share the model / link for the speaker you chose (single speaker)? I'm considering a similar approach as you.

**Reply by luisete79** [reply]
Sure, you can see a picture here:
https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/2115#discussioncomment-9490284
The speaker belonged to an old TV that broke.
